lib/msal-node/docs/caching.md

```diff
@@ -38,25 +38,27 @@ cca.acquireTokenSilent(silentTokenRequest).then((response) => {
 
-## Cache in memory
+## In-memory cache
 
-If you choose not to persist the cache, the `TokenCache` interface is still available to access the tokens cached in memory. The lifetime of in-memory cache is the same as MSAL instance. If the MSAL instance restarts, the cache is erased when the process lifecycle finishes. We recommend persisting the cache with encryption for all real life applications both for security and desired cache longevity.
+MSAL maintains an in-memory cache. The in-memory cache is the single source of truth for all MSAL activities involving tokens and accounts. The lifetime of in-memory cache is the same as the MSAL application object. If the process using MSAL restarts, the cache is erased when the process lifecycle finishes. This affects user scenarios as users will have to re-authenticate, and although the browser will do most some of the work, the end user experience suffers. Service to service scenarios also suffer because getting a token from AAD involves HTTP requests, and is much slower than getting a token from a cache.
 
-> :warning: Please note that the default in-memory cache is not scalable for server-side applications and performance will degrade after holding a few 100 tokens in cache. For web app and web API scenarios, this approximates to serving a few 100 users. For daemon app scenarios using client credentials grant to call other apps, this means a few 100 tenants. See [performance](#performance-and-security) below for more.
+Note that the in-memory cache is not scalable for server-side applications and performance will degrade after holding a few 100 tokens in cache. For web app and web API scenarios, this approximates to serving a few 100 users. For daemon app scenarios using client credentials grant to call other apps, this means a few 100 tenants. See [performance](#performance-and-security) below for more.
 
-## Persistence
+> :warning: We recommend persisting the cache with encryption for all production applications both for security and desired cache longevity. If you choose not to persist the cache, the [TokenCache](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_node.tokencache.html) interface is still available to access the cached entities.
 
-MSAL Node fires events when the cache is accessed, apps can choose whether to serialize or deserialize the cache. This often constitutes two actions:
+## Persistent cache
 
-1. Deserialize the cache from disk to MSAL's memory before accessing the cache
-2. If the cache in memory has changed, serialize the cache back
+MSAL Node fires events when the in-memory cache is accessed and apps can choose whether to serialize or deserialize the cache (see: [TokenCacheContext](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_common.tokencachecontext.html)) to persistence (e.g. a file, a SQL database and etc.). This constitutes two actions:
```

Comment:
Do we need to mention serialization? It is just a technique to ensure cache integrity. We can just say `persist` here. If you believe you need the advanced users to know the details for performance reasons, I prefer to separate the description and expand on why we settled on a `json` format to persist cache and we apply serialization and de-serialization to achieve the same.

Reply:
I see why you may have done this, as the API name consists of it. However, the above still applies.

Reply:
Sure thing, used save/load instead to be agnostic about this -there's enough hints elsewhere about using the serialization technique